mage_integrations/mage_integrations/tests/sources/test_base.py

```diff
@@ -292,7 +292,7 @@ def test_sync(self):
         catalog.get_selected_streams.return_value = build_sample_streams_catalog_entries()
         source = Source()
         with patch.object(source, 'process_stream', return_value=None) as mock_process_stream:
-            with patch.object(source, 'sync_stream') as mock_sync_stream:
+            with patch.object(source, 'sync_stream', return_value=dict()) as mock_sync_stream:
```

Comment:
return value should be a number https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/sources/base.py#L377